doom-modeline-segments.el

```diff
@@ -1359,36 +1359,38 @@ of active `multiple-cursors'."
 (doom-modeline-def-segment bar
   "The bar regulates the height of the mode-line in GUI."
   (when doom-modeline-bar
+    (unless (and doom-modeline--bar-active doom-modeline--bar-inactive)
+      (let ((width doom-modeline-bar-width)
+            (height (max doom-modeline-height
+                         (doom-modeline--font-height))))
+        (when (and (numberp width) (numberp height))
+          (setq doom-modeline--bar-active
+                (doom-modeline--make-image 'doom-modeline-bar width height)
```

Comment:
This doesn't look right: it's undoing the rename of doom-modeline--make-image to doom-modeline--create-bar-image (looks like something went wrong merging master). I've sent #444 to fix it.

Reply:
It's a merge issue. I've already fixed it. 3b39376